packages/core/src/migration/index.ts

```diff
@@ -0,0 +1,134 @@
+import { readdir, readFile } from 'fs/promises';
+import path from 'path';
+
+import { LogtoConfig } from '@logto/schemas';
+import { conditionalString } from '@silverhand/essentials';
+import chalk from 'chalk';
+import { DatabasePool, sql } from 'slonik';
+import { raw } from 'slonik-sql-tag-raw';
+
+import {
+  databaseVersionKey,
+  logtoConfigsTable,
+  logtoConfigsTableFilePath,
```

Comment:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/14722250/191243625-5775a74c-bfb9-4e39-89d3-555736cce437.png">

we can use the generated constants in schemas? just like other queries in `core`:

`packages/core/src/queries/application.ts`

<img width="526" alt="image" src="https://user-images.githubusercontent.com/14722250/191243817-c3cb2d65-c9ea-484c-b096-935eb7d7358d.png">

Reply:
updated